consensus/src/epoch_manager.rs

```diff
@@ -146,18 +168,22 @@ impl EpochManager {
             self_sender,
             network_sender,
             timeout_sender,
-            quorum_store_to_mempool_sender,
+            quorum_store_to_mempool_tx,
             commit_state_computer,
             storage,
             safety_rules_manager,
             reconfig_events,
-            commit_notifier,
+            data_manager,
             buffer_manager_msg_tx: None,
             buffer_manager_reset_tx: None,
             round_manager_tx: None,
             round_manager_close_tx: None,
             epoch_state: None,
             block_retrieval_tx: None,
+            quorum_store_storage: Arc::new(QuorumStoreDB::new(path)),
+            num_network_workers_for_fragment: 2,
+            quorum_store_msg_tx_vec: Vec::new(),
```

Comment:
This should be made into a proper struct. index 0 and 1 are reserved for batch and signed_digest, which is ugly. The struct can provide `fn send_signed_digest`, `fn send_batch`, and `fn send_fragment(idx)`

Reply:
And `num_network_workers_for_fragment` can be internal to the struct